packages/slice-machine/pages/slices.tsx

```diff
@@ -145,21 +167,64 @@ const SlicesIndex = ({ env }: { env: Environment }) => {
                 </Box>
               )}
             </Box>
-          ) : null}
-          <Box>
-            {libraries &&
-              libraries.map((maybelib: LibraryState | undefined, i) => {
+          )}
+          <Box
+            sx={{
+              flex: 1,
+              display: "flex",
+              flexDirection: "column",
+            }}
+          >
+            {sliceCount == 0 ? (
+              <Box
+                sx={{
+                  flex: 1,
+                  alignItems: "center",
+                  flexDirection: "column",
+                  justifyContent: "center",
+                  display: "flex",
+                  textAlign: "center",
+                }}
+              >
+                <Heading sx={{ mb: 4 }}>Create your first Slice</Heading>
+                <Text sx={{ mb: 4 }}>
+                  Click the + button on the top right to create the first slice
+                  of your project. <br />
+                  <br /> It will be stored locally. You will then be able to
+                  push it to Prismic.
+                </Text>
+                <Button onClick={() => setIsOpen(true)}>
+                  {data.loading ? (
```

Comment:
data.loading && (i'm pointing it out because i've seen it iin your PRs rather than the `?` 😄 )

Reply:
Good catch,